components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1673,10 +1723,12 @@ void RewardsServiceImpl::OnLoadedState(
     const std::string& value) {
   if (!Connected())
     return;
-  if (value.empty())
+  if (value.empty()) {
```

Comment:
similar to above, I'd rather do `RecordWalletCreatedMetrics` before this line and pass in `value` and `this`

Reply:
IMO in this case adding `friends` will only reduce readability and reasoning about this code

Reply:
I disagree for the reason here https://github.com/brave/brave-core/pull/3242#discussion_r330325793
this file is already basically unreadable and we need to break the code up into smaller, more focused pieces

Reply:
IMO we are talking about the same thing - refactoring and splitting godlike services like rewards&ads. This is is apparently the root of the problem.

Reply:
@iefremov The good news is that we are gradually separating concerns when working in those areas, if there is anything we can help with just let us know